### DISCUSSION TITLE
Cache files

### DISCUSSION THREAD

**Discussion by msisynotify** [iterate-ch/cyberduck#14708] [Q&A]
I have both Mountainduck and Cyberduck and have paid for both.  On my windows machine, Mountainduck is storing 95GB of cache and Cyberduck 80GB .  

For Mountainduck, the choices are restricting cache file size or number of days - however, the limit is 10gb and yet there are 95gb of cache files. so that is not an answer.  The days option is also no good, as somehow all the files have a created and modified date of today, even though I accessed at most 4 files today.

For Cyberduck, I see no information about cache files at all in any help or discussion or setting or configuration.  The files are just there with no known way to get rid of the,.  It cannot be that this software needs 175gb to run.  I need a way to clean up the cache and then have it stay to a reasonable size for each.

Please advice.

thx

**Comment by AliveDevil** [comment]
Cyberduck shouldn't take up any additional space, than is required for configured bookmarks.
Where did you see, that Cyberduck takes up 80 GB of storage? Microsoft Store installed, or standalone from website?

Mountain Duck clears its sync cache once every day. Files marked "keep offline" aren't checked for clearing up space.

**Reply by msisynotify** [reply]
i will need to wait for Mountainduck for a full day to pass
For Cyberduck, here is a path to the place taking all the space - C:\Users\seaco\AppData\Local\Cyberduck\Cache\734724d0-54ac-4809-ae47-9bbf99caa230\4\d\2A\2EJGFZZZSI547RXZCEBZ5NOVFD7GHK

the "d" folder has many folders in it with 2 char alphanumeric names and each of those folders has folders like the one here that look like encrypted file names from a vault - in fact, this looks very much like a Cryptomator vault structure.  

The dates on these are the dates from the last  time Cyberduck was open, so these are definitely linked to using Cyberduck
